error: unexpected character after line continuation
   ╭─[ test.asm:2:10 ]
   │
 2 │     def x = \0
   │             ┬  
   │             ╰── there can only be whitespace between this backslash and the end of the line
   │ 
 5 │     m hello, world
   │     ┬  
   │     ╰── macro called here
───╯
error: macro argument #0 doesn't exist
   ╭─[ test.asm:3:10 ]
   │
 3 │     def y = \<0> + 42
   │             ──┬─  
   │               ╰─── macro argument 0 doesn't exist
   │ 
 5 │     m hello, world
   │     ┬  
   │     ╰── macro called here
   │ 
   │ Help: macro arguments are 1-indexed: the first macro argument is `\1`
───╯
2 errors generated.
